sunkit_instruments/suvi/tests/test_suvi.py

```diff
@@ -24,15 +26,15 @@ def test_suvi_despiking_nc(L1B_NC):
 
 
 def test_get_response_nc(L1B_NC):
-    l1b_nc_response = suvi.get_response(L1B_NC)
+    l1b_nc_response = suvi.get_response(L1B_NC, spacecraft=16, ccd_temperature=-60.0 * u.deg_C, exposure_type="long")
     assert l1b_nc_response["wavelength_channel"] == 171
```

Comment:
Somewhere either in these tests or new ones, we need to check that the output is ok as well.

Reply:
Can i get the paths for the L1B_FITS and L1B_NC files?

Reply:
I assume they are in the repository somewhere. I would check where `L1B_NC` is defined, that should lead you to the files.

Reply:
ok please.

Reply:
I am not sure I follow.